main/test/ca/uwaterloo/flix/tools/TestFlixPackageManager.scala

```diff
@@ -71,12 +75,16 @@ class TestFlixPackageManager extends FunSuite {
       }
 
       val path = Files.createTempDirectory("")
-      FlixPackageManager.installAll(manifest, path)(System.out)
+      FlixPackageManager.installAll(manifest, path)(System.out) match {
+        case Ok(l) => l.head.endsWith("magnus-madsen\\helloworld\\ver1.0.0\\helloworld.fpkg") &&
```

Comment:
These paths are filesystem dependent, right?

Reply:
No these should be the same no matter where - that is why I only check the ending of the strings

Reply:
Also the `\\`?

Reply:
I did not consider that - yes, then it is file system dependent. I will fix that